src/extension.ts

```diff
@@ -40,7 +40,7 @@ async function init(context: vscode.ExtensionContext, git: ApiImpl, gitAPI: GitA
 	Logger.appendLine('Git repository found, initializing review manager and pr tree view.');
 
 	vscode.authentication.onDidChangeSessions(async e => {
-		if (e['github']) {
+		if (e.provider.id === 'github') {
```

Comment:
Do you also need to update the VS Code verion in the package.json?

Reply:
Yes, thank you